build.gradle

```diff
@@ -186,6 +202,7 @@ configure(subprojects) { project ->
   // dependencies that are common across all java projects
   dependencies {
 	compile "io.projectreactor:reactor-core:$reactorCoreVersion"
+	  compile 'org.jacoco:org.jacoco.agent:0.8.3'
```

Comment:
we've never had to add the agent nor even the core jars in our projects before. can you clarify why this is needed?